tests/resspect/test_feature_handling_utils.py

```diff
@@ -0,0 +1,73 @@
+import mongomock
+import os
+import pandas as pd
+from pathlib import Path
+import tempfile
+from unittest.mock import patch
+
+from resspect.feature_handling_utils import *
+
+_TEST_DATA_DIR = Path(__file__).parent.parent.parent / "data" / "tests"
+_TEST_MONGO_URI = "mongodb+srv://resspect:0000000V@resspectcluster0.agzec.mongodb.net/"
```

Comment:
@maxwest-uw is there any admin/ownership/permissions here that needs to be handed off to the RESSPECT folks? I'm assuming this is a free tier hosted instance, and just want to make sure that Amanda/the next owner can do what they need to with it.